mpyl-reporter.py

```diff
@@ -20,55 +20,60 @@
 
 
 def main(logger: Logger):
-    run_result_file = Path(BUILD_ARTIFACTS_FOLDER) / "run_result"
-
-    if not run_result_file.is_file():
-        logger.info(f"Run result file {run_result_file} not found. Nothing to report.")
+    run_result_files = list(Path(BUILD_ARTIFACTS_FOLDER).glob(RUN_RESULT_FILE_GLOB))
+    if len(run_result_files) == 0:
+        logger.info(
+            f"Run result file(s) {RUN_RESULT_FILE_GLOB} not found. Nothing to report."
+        )
         sys.exit()
 
-    with open(run_result_file, "rb") as file:
-        run_result: RunResult = pickle.load(file)
-
-        accumulator = ReportAccumulator()
-        config = parse_config("mpyl_config.yml")
-        properties = parse_config("run_properties.yml")
-        run_properties = construct_run_properties(config=config, properties=properties)
-
-        commit_check = CommitCheck(config=config, logger=logger)
-        accumulator.add(commit_check.send_report(run_result))
+    config = parse_config("mpyl_config.yml")
+    properties = parse_config("run_properties.yml")
+    run_properties = construct_run_properties(config=config, properties=properties)
+    run_result: RunResult = RunResult(run_properties=run_properties)
```

Comment:
For my understanding: what's the purpose of `ReportAccumulator` now that we've done this change? It looks like we're accumulating all `run_resuilt-*.pickle` files into a single `RunResult`, and then accumulating this again into an `ReportAccumulator`. Is there an opportunity to simplify this setup?

Reply:
Yeah that makes sense, thanks for explaining! 👍